google/cloud/ndb/_gql.py

```diff
@@ -93,18 +93,29 @@ class GQL(object):
     _limit = -1
     _hint = ""
 
-    def __init__(self, query_string, _app=None, _auth_domain=None, namespace=None):
+    def __init__(
+        self,
+        query_string,
+        _app=None,
+        _auth_domain=None,
+        namespace=None,
+        *,
+        database: str = None
+    ):
         """Parses the input query into the class as a pre-compiled query.
 
         Args:
             query_string (str): properly formatted GQL query string.
-            namespace (str): the namespace to use for this query.
+            database (str): The database to use for this query. Defaults to the client's value.
```

Comment:
I'd put namespace before database on this list, to reflect the order in the definition.